addons/web_dashboard/static/src/js/dashboard.js

```diff
@@ -0,0 +1,264 @@
+        var menu_id = $(e.currentTarget).attr('data-menu-id');
+        if (this.planner && this.planner.menu_id[0] == menu_id) {
+             this.dialog.do_show();
+        }else{
+            this.setup_planner(menu_id);
+        }
+    },
+    setup_planner: function(menu_id){
+        var self = this;
+        this.planner = self.planner_by_menu[menu_id];
+        if (this.dialog) {
+            this.dialog.destroy()
+        }
+        this.dialog = new PlannerDialog(this, this.planner);
+        this.dialog.on("planner_progress_changed", this, function(percent) {
+            self.planner.progress = percent;
+            self.update_planner_progress();
+        });
+        this.dialog.appendTo(webclient.$el).then(function() {
+            self.dialog.do_show();
+        });
+    }
+});
+
+var DashboardGift = Widget.extend({
+    template: 'DashboardGift',
+    events: {
+        'click .tw_share': 'share_twitter',
+        'click .fb_share': 'share_facebook',
+        'click .li_share': 'share_linkedin',
+    },
+    init: function(parent, data){
+        this.data = data;
+        this.parent = parent;
+        this.share_url = _.str.sprintf('http://www.odoo.com/?referral=%s', this.data.user_email);
+        this.share_text = encodeURIComponent("#IamUsingOdoo, an Open-Source Web App that manages my Sales, Projects, Accounting, Website, Warehouse, Shop and more");
+        var ZeroClipboard = window.ZeroClipboard;
+        ZeroClipboard.config({swfPath: location.origin + "/web/static/lib/zeroclipboard/ZeroClipboard.swf" });
+        return this._super.apply(this, arguments);
+    },
+    start: function(){
+        var self = this;
+        return $.when(this._super()).then(function() {
+            var client = new ZeroClipboard(self.$('.o_dashboard_sharearea'));
+            client.on("ready", function(readyEvent) {
+                client.on("aftercopy", function(event) {
+                    $(event.target).hide();
+                    self.$('.o_clipbord_success_alert').show();
+                });
+            });
+            new ZeroClipboard(self.$('.o_clipbord_success_alert'));
+        });
+    },
+    share_twitter: function(){
+        var popup_url = _.str.sprintf( 'https://twitter.com/intent/tweet?tw_p=tweetbutton&text=%s %s',this.share_text,this.share_url);
+        this.sharer(popup_url);
+    },
+    share_facebook: function(){
+        var popup_url = _.str.sprintf('https://www.facebook.com/sharer/sharer.php?u=%s', encodeURIComponent(this.share_url));
+        this.sharer(popup_url);
+    },
+    share_linkedin: function(){
+        var popup_url = _.str.sprintf('http://www.linkedin.com/shareArticle?mini=true&url=%s&title=I am using odoo&summary=%s&source=www.odoo.com', encodeURIComponent(this.share_url), this.share_text);
+        this.sharer(popup_url);
+    },
+    sharer: function(popup_url){
+        window.open(
+            popup_url,
+            'Share Dialog',
+            'width=626,height=436');
```

Comment:
In chrome if i don't give height-width it open it in new tab, instead window. 
If i give height-width 1-1 it is like twitter window in screenshot. 
I just gave approx width and height (yes randomly !!) because facebook and linkedin changes window height according to content. I will change it with some proper number like 600*400  :smile:  
![screenshot from 2015-08-27 18 15 45](https://cloud.githubusercontent.com/assets/7629089/9520805/e9f934f4-4ce7-11e5-9249-87361f294941.png)